### PR TITLE
fixing few typos

### PR DESCRIPTION
**Pull request by nikpalyi**

### CODE REVIEW

**Review by mcclellangg — APPROVED** [review]
Looks good. I always thought that Time Zone could be spelled either way, but apparently timezone (like we had is more US favored). Considering the nature of the app makes sense to switch it 👍 .
https://english.stackexchange.com/questions/3934/time-zone-vs-timezone

File: README.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ Welcoming page displaying brief information about the site, and asking users to
 
 ### Registration
 
-_**/pages/acount/Signup.js**_
+_**/pages/account/Signup.js**_
 Page that consists of a registration form _/components/Form.js_ for new users, asking them to provide the following info:
```
Comment (orsolyalukacs) [inline README.md:24]: It's not a typo. This is specifically how the project structure looks like, so I wouldn't change it. 
Thanks!

Comment (orsolyalukacs) [inline README.md:24]: Thanks!

**Review by orsolyalukacs — APPROVED** [review]

**Comment by orsolyalukacs** [comment]
@nikpalyi Merged, thanks!
Next time, please send a PR to the dev branch :) 
We are following this workflow: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

**Comment by orsolyalukacs** [comment]
@nikpalyi considering we would need to re-write "timezone" to "time zone" everywhere in the codebase and it might be better to keep it "timezone" according to US English, could you please re-send the PR with only the other changes only?
Thank you!

**Comment by nikpalyi** [comment]
Thanks for the constructive feedbacks. Actually the all look good originally , I wanted to close the PR.

**Pull request merged**